platforms/js/build_js.py

```diff
@@ -194,11 +196,25 @@ def build_doc(self):
     parser.add_argument('--skip_config', action="store_true", help="Skip cmake config")
     parser.add_argument('--config_only', action="store_true", help="Only do cmake config")
     parser.add_argument('--enable_exception', action="store_true", help="Enable exception handling")
-    args = parser.parse_args()
+    parser.add_argument('--cmake_option', action='append', help="Adds the ability to modify the list of modules")
```

Comment:
please modify description. This options are not only for modules

Reply:
May be writ like ` In each section [], we describe the functions and classes `